addons/l10n_pe/data/account_fiscal_position_template_data.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
@nhomar Best to have an empty fiscal position with country_id Peru auto_apply True before this one.  And you can put this one in auto_apply as well.

Reply:
done